template/src/features/counter/fakeApi.js

```diff
@@ -0,0 +1,4 @@
+export const fakeAsyncCall = async amount => {
+  await new Promise(resolve => setTimeout(resolve, 1000));
+  return amount;
+};
```

Comment:
I know this is probably just here for demonstrational purposes, but something a little more real-world would be nice to showcode best practices. For example this could use `fetch` to call out to https://example.com/ or https://jsonplaceholder.typicode.com/.

Reply:
It seems odd to call an api and then ignore what it returns, even for the purposes of demonstration.

Reply:
Could make a call to a random number api and increase the counter by what it returns? e.g. https://www.random.org/clients/http/api/

Reply:
To be clear I'm suggesting actually using some fake data, not just making a request to make it async and ignoring the response.

Reply:
Done - using https://www.random.org/clients/http/api/ @nickmccurdy @BenLorantfy